homeassistant/components/enocean/__init__.py

```diff
@@ -10,28 +13,42 @@
 from .const import DATA_ENOCEAN, DOMAIN, ENOCEAN_DONGLE
 from .dongle import EnOceanDongle
 
+_LOGGER = logging.getLogger(__name__)
+
 CONFIG_SCHEMA = vol.Schema(
     {DOMAIN: vol.Schema({vol.Required(CONF_DEVICE): cv.string})}, extra=vol.ALLOW_EXTRA
 )
 
 
+# component setup
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the EnOcean component."""
+
+    if not hass.data.get(DOMAIN):
+        services.async_setup_services(hass)
```

Comment:
See above suggestion:
```suggestion
        async_setup_services(hass)
```